main.py

```diff
@@ -409,7 +409,9 @@ def index(title, language="en"):
         # is_german_user = response and response.country.iso_code == 'DE'
 
         return make_response(render_template(
-            "index-python.html" if (language == "en" and domain_data["language"] == "python") else "index.html",
+            "index-python.html" if (language == "en" and domain_data["language"] == "python") else \
+            "index-js.html" if (language == "en" and domain_data["language"] == "javascript") else \
```

Comment:
I would prefer to remove duplications of index.html. python is special because datacamp has their own method of code execution.